src/components/ConnectWallet.tsx

```diff
@@ -16,7 +16,7 @@ function ConnectWallet({ onClose }: Props): ReactElement {
   const { activate } = useWeb3React()
 
   return (
-    <div className="connectWallet">
+    <div className="connectWallet" data-testid="connectWallet">
```

Comment:
Step in the right direction. I'd make it a little more descriptive i.e. `connectWalletContainer`